go/kbfs/libkbfs/conflict_resolver.go

```diff
@@ -3324,6 +3325,57 @@ func (cr *ConflictResolver) recordFinishResolve(
 	err = serializeAndPutConflicts(cr.config, db, key, conflictsSoFar)
 }
 
+func (cr *ConflictResolver) makeDirtyBcache(
+	ctx context.Context, kmd KeyMetadata) (
+	dirtyBcache DirtyBlockCacheSimple, cleanupFn func(context.Context),
+	err error) {
+	var dbc *DiskBlockCacheLocal
+	if cr.config.IsTestMode() {
+		// Enable the disk limiter if one doesn't exist yet.
+		_ = cr.config.(*ConfigLocal).EnableDiskLimiter(os.TempDir())
+
+		dbc, err = newDiskBlockCacheLocalForTest(
+			cr.config, syncCacheLimitTrackerType)
+		if err != nil {
+			return nil, nil, err
+		}
+		cleanupFn = dbc.Shutdown
+	} else {
+		tempDir, err := ioutil.TempDir(cr.config.StorageRoot(), "kbfscr")
```

Comment:
There's already a `kbfs_conflicts` for the CR levelDB - maybe `kbfscr` should get a slightly clearer name? [and maybe the other one should too..]

Reply:
Ok, changing it to "kbfs_conflict_disk_cache".